qipr_approver/approver/templates/approver/dashboard_su.html

```diff
@@ -83,7 +83,6 @@ <h1 class="question__header font__opensans"><i class="icon__admin medium materia
                                 <span>|</span>
 
                                 <!-- ARCHIVE BUTTON -->
-                            {% if request.session.su %}
```

Comment:
two reasons: one, we don't put su into the request anymore since users are correctly passed now. Second, this page cannot be reached if you are not a super user